config.yml

```diff
@@ -37,26 +36,27 @@ janssen_pooled_real: &janssen_default_real
 janssen_na_real: &janssen_default_real
   <<: *janssen_default_real
   subset_variable: Region
-  subset_value: Northern America
+  subset_value: 0
```

Comment:
Are you confident that this numeric labeling is stable? I get nervous about how factors/characters get cast to numerics, so just wanted to double check.